tests/optimizers/bohb_test.py

```diff
@@ -250,3 +249,81 @@ def test_bohb_with_none_min_samples_in_model():
         min_samples_in_model=None,
     )
     assert opt.config_sampler.min_samples_in_model == 3
+
+
+def test_minimize_quadratic():
+    paramspace = ps.ParameterSpace()
+    paramspace.add(ps.ContinuousParameter("p1", (-2.0, 2.0)))
+    opt = BOHB(
+        paramspace,
+        Objective("loss", False),
+        min_fidelity=1.0,
+        max_fidelity=9.0,
+        num_iterations=10,
+        seed=42,
+    )
+
+    evals = []
+    for _ in range(60):
+        es = opt.get_evaluation_specification()
+        e = es.create_evaluation(objectives={"loss": es.configuration["p1"] ** 2})
+        evals.append(e)
+        opt.report(e)
+
+    best_eval = min(evals, key=lambda e: e.objectives["loss"])
+    assert abs(best_eval.objectives["loss"]) < 0.1
```

Comment:
Not sure if that is an ideal `assert`, as this regret bound might not be achieved in the first 9 trails.